requirements.txt

```diff
@@ -1,3 +1,4 @@
 pyglet==2.0.9
 tk
 numpy
+pre-commit
```

Comment:
Could you add version numbers here? This just makes dependency resolution faster and stops any problems occuring because of incompatible packages in the future